content/en/users/check-in/vos/_index.md

```diff
@@ -91,9 +91,8 @@ The VO enrollment URL can be found through the EGI Check-in Registry:
 ### Managing VO groups
 
 VO groups can only be created by Check-in platform administrators. Please
-contact [Checkin Support](mailto:checkin-support@mailman.egi.eu) indicating the
-following information for every (sub)group that you need to add/remove to/from
-your VO:
+contact `checkin-support` `<AT>` `mailman.egi.eu` indicating the following
```

Comment:
So here and there we have been using directly the email with a mail-to, usually our addresses are already quite easily findable on the net, not sure it's totally required to do a trick like this to try to avoid being spammed.

Reply:
Or @nikosev do you feel you have been receiving too many spam that wasn't blocked by mailman and so on?

Reply:
@gwarf We get on average 5 spam mails per month on this email address, which create automatically GGUS tickets. But if you think it is pointless to try to protect it, then I can revert this change

Reply:
OK, fine with me to keep it like this, just not sure how effective this protection will be.